packages/terra-action-header/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -0,0 +1,45 @@
+@import '~terra-mixins/lib/Mixins';
+
+:global {
+
+  // ---------------------- Shared ----------------------- //
+
+  --terra-action-header-background-color: #141719;
+  --terra-action-header-border-bottom: 1px solid #181d1b;
+  --terra-action-header-border-left: none;
+  --terra-action-header-border-right: none;
+  --terra-action-header-border-top: none;
+  --terra-action-header-box-shadow: none;
+  --terra-action-header-color: #b2b5b6;
+
+  // -------------------- Version 2.16.0+ -------------------- //
+
+  // Icons
+  @include terra-inline-svg-var('--terra-action-header-back-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M48 21H10.6L27.3 3.9 23.5.1 0 24l23.5 23.9 3.8-3.8L10.6 27H48z"></path></svg>');
+  @include terra-inline-svg-var('--terra-action-header-close-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z"></path></svg>');
+  @include terra-inline-svg-var('--terra-action-header-maximize-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M17.7 26.1L6.9 36.9 0 30v18h18l-6.9-6.9 10.8-10.8zM48 0H30l6.9 6.9-10.8 10.8 4.2 4.2 10.8-10.8L48 18z"></path></svg>');
+  @include terra-inline-svg-var('--terra-action-header-minimize-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M10.8 32.9L0 43.7 4.2 48l10.9-10.8 6.9 6.9V26H3.9zM48 4.2L43.8 0 32.9 10.8 26 3.9V22h18.1l-6.9-6.9z"></path></svg>');
+  @include terra-inline-svg-var('--terra-action-header-next-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M24 37.7L0 14.2l3.8-3.9L24 30l20.2-19.7 3.8 3.9z"></path></svg>');
+  @include terra-inline-svg-var('--terra-action-header-previous-background-image', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M24 10.3l24 23.5-3.8 3.9L24 18 3.8 37.7 0 33.8z"></path></svg>');
+
+  
+  // -------------------- Version 2.x -------------------- //
+
+  // Icons
+  @include terra-inline-svg-var('--terra-action-header-back-background', '<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#c5c5c6" d="M48 21H10.6L27.3 3.9 23.5.1 0 24l23.5 23.9 3.8-3.8L10.6 27H48z"></path></svg>');
```

Comment:
How does adding old version numbers provide value?  If this is going to release with (for example) terra-action-header 2.34.0, in what situation does having the styling for 1.x help?   Are there any situations it hurts?

Reply:
I had a similar query, but after a discussion with @ryanthemanuel we came to a conclusion to port all the theme files and variables as it is present in the theming repo and then maybe later have a separate issue to remove the legacy variables after assessing their impact.

Reply:
This has now been addressed here : https://github.com/cerner/terra-core/pull/2758/commits/28a7c8dc409e0e443329574f2c3b8e220c7fa250